Guides/Generating your models.md

```diff
@@ -0,0 +1,144 @@
+# Generating your IGListDiffable models
+
+With the `IGListDiffable` plugin for [remodel by facebook](https://github.com/facebook/remodel), you can automatically generate models conforming to the `IGListDiffable`.
+
+This will automatically implement `hash`, `isEqual:` and `description`, as well as `diffIdentifier` and `isEqualToDiffableObject:` for you. Remodel is also capable to generate additional code, like conforming to `NSCoding` or additional Builder classes for your model object. It will make creating and updating models much easier, faster and safer.
+
+In `/remodel-plugin`, you can find the source files to build the `IGListDiffable` plugin locally.
+
+### 1. Remodel installation (if you don't have it yet)
+
+Please follow the installation instructions on https://github.com/facebook/remodel.
```

Comment:
nit: how about the following:

> Please follow the [installation instructions](https://github.com/facebook/remodel).

Reply:
I wanted to be verbose about the other repo. But sure.